src/Build.UnitTests/BackEnd/TaskBuilder_Tests.cs

```diff
@@ -878,12 +878,13 @@ Task IRequestBuilderCallback.BlockOnTargetInProgress(int blockingRequestId, stri
 
         #endregion
 
-        /*********************************************************************************
-         * 
-         *                                     Helpers
-         * 
-         *********************************************************************************/
+/*********************************************************************************
+ * 
+ *                                     Helpers
+ * 
+ *********************************************************************************/
 
+#if FEATURE_CODEDOM
```

Comment:
This feels fishy. Why was it not needed before?

Reply:
It wasn't "needed", but the only callers are `ValidateDefiningProjectMetadataOnTaskOutputs` and `ValidateDefiningProjectMetadataOnTaskOutputs_LegacyItems` which are also behind this ifdef.

IE. it's currently dead code when `!FEATURE_CODEDOM`.

Reply:
Oh, I figured that pattern out later but maybe didn't see it here.